### ISSUE TITLE
Json cannot be saved

### ISSUE THREAD

**Issue by zenControl4** [hufftheweevil/node-red-contrib-things#27]
Sorry, I don’t speak English, this is from Google Translate.
<img width="596" alt="WX" src="https://user-images.githubusercontent.com/75163021/112744470-f74f3700-8fd2-11eb-8417-9992e24e250f.png">
As shown in the figure, JSON cannot be saved normally.

**Comment by zenControl4** [comment]
I found that both "state" and "commands" have this problem, but "props" does not.
<img width="598" alt="props" src="https://user-images.githubusercontent.com/75163021/113103252-a0936880-9231-11eb-8ffb-c7052d48e02b.png">
<img width="619" alt="commands" src="https://user-images.githubusercontent.com/75163021/113103328-bb65dd00-9231-11eb-9f50-47b58ad3c7c4.png">
<img width="619" alt="state" src="https://user-images.githubusercontent.com/75163021/113103343-bf91fa80-9231-11eb-8c7f-e143e60ae4d2.png">


So I checked the source code and found the difference in ui-utils.js.
The  THI.addProp function is like this：
https://github.com/hufftheweevil/node-red-contrib-things/blob/0c00aa58af4d68c8602fc27ac334f01fe582f6ae/lib/ui-utils.js#L466

In THI.addProp, the data is converted by the typedToString function, but not in THI.addState and THI.addCommands.
Adding the typedToString function to THI.addState and THI.addCommands, JSON can be saved and read normally on my computer.
